pkg/service/aws/volume.go

```diff
@@ -295,15 +275,15 @@ func (svc awsController) CreateSnapshotAndDelete(ctx context.Context, req *proto
 	//calling aws sdk method to delete volume
 	//ec2.DeleteVolumeOutput doesn't contain anything
 	//hence not taking response
-	_, err = ec2Client.DeleteVolume(inputDelete)
+	_, err = ec2Client.DeleteVolumeWithContext(ctx, inputDelete)
```

Comment:
Wait for volume to delete here as well or not use the `WithContext` method

Reply:
With context can only handle request, it cannot propogates to aws.